package.json

```diff
@@ -18,9 +18,15 @@
     "@react-native-firebase/storage": "^6.7.1",
     "formik": "^2.1.4",
     "prop-types": "^15.7.2",
+<<<<<<< HEAD
```

Comment:
the arrows/lines indicate that there was a merging conflict, you have to determine which ones should be added or not when merging